### PR TITLE
base image update and s3 aws config

### PR DESCRIPTION
**Pull request by bhayden53**
goes with calcloud pr https://github.com/spacetelescope/calcloud/pull/75

### CODE REVIEW

**Review by jaytmiller — APPROVED** [review]
LGTM

**Comment by bhayden53** [comment]
waiting for approval on the accompanying calcloud PR before merge

**Pull request merged**
